src/templates/UserList/UserList.tsx

```diff
@@ -0,0 +1,61 @@
+import { CustomTableToolbarProps, HeadCell, SortingRule } from '../../components/CustomTable/CustomTable.types';
+import { CustomTable } from '../../components/CustomTable/CustomTable';
+import { SortingDirection } from '../../common/utils/SortingDirection';
+import { User } from '../../common/types/componentTypes.types';
+import { UserListToolbar } from './UserListToolbar';
+import { useState } from 'react';
+
+export function UserList() {
+  const [sortingConfig, setSortingConfig] = useState<SortingRule[]>(buildCustomTableSortingConfig());
+  const [selectedRowsIds, setSelectedRowsIds] = useState<readonly string[]>([]);
+  const [page, setPage] = useState(0);
+  const [initialRows, setRows] = useState<User[]>([]);
+  const defaultRowsPerPage = 10;
+  const [rowsPerPage, setRowsPerPage] = useState(defaultRowsPerPage);
+  const url = '../src/templates/UserList/testUsers.json';
+
+  function buildCustomTableHeaderCells(): HeadCell<keyof User>[] {
+    const headerCells: HeadCell<keyof User>[] = [];
+    // eslint-disable-next-line sort-keys
+    const row = { id: '', email: '', name: '', password: '', roles: [], roomNumber: 0, surname: '' } as User;
+
+    Object.keys(row).forEach((key) => {
+      headerCells.push({ id: key as keyof User, label: key });
```

Comment:
I think label should be translated and for sure shoudln't be equal to key

Reply:
this should be fixed in [08f6a45](https://github.com/dorm-social-room-manager/dsrm-frontend/pull/38/commits/08f6a4533a8460a854d4bdbe56059b6997dbc9ec)